content/swiftui/concepts/navigation/navigation.md

```diff
@@ -0,0 +1,164 @@
+
+## NavigationStack with Lists
+
+One of the common examples of using `NavigationStack` is with `Lists`. For example, there is a list of names in the view. The goal is to be able to navigate to another view by pressing the element in the list. This can be achieved by wrapping the `List` in a `NavigationStack` :
+
+```Swift
+struct Doc: View {
+    let Names = ["Omar","Adam","Zack"] // this is the list
+    var body: some View {
+        NavigationStack{
+            List(Names,id:\.self){ name in // List iterates throw the Names array, every iteration the element is stored in name.
+                NavigationLink {
+                    Text(name) // desired view destination
+                } label: {
+                    Text(name)  
+                }
+            }
+            
+        }
+    }
+}
+```
+
+This code will output:
+
+![This is a gif that shows how to use NavigationStack with Lists](https://raw.githubusercontent.com/Codecademy/docs/main/media/NavigationStack-with-List.gif)
+
+Those are just basic examples for `NavigationStack`, there are no limits on how to use it.
+
+## NavigationStack Modifiers
+
+There are some basic modifiers to use with `NavigationStack`. The first one is `.navigationTitle`. This is used to be able to declare the title of the view. It looks like this:
+
+```Swift
+struct Doc: View {
+    let Names = ["Omar","Adam","Zack"]
+    var body: some View {
+        NavigationStack{
+            List(Names,id:\.self){ name in
+                NavigationLink {
+                    Text(name) // desired view destination
+                } label: {
+                    Text(name)
+                }
+            }
+            .navigationTitle("Names")
+        }
+    }
+}
+```
+
+This code will output:
+
+![This picture shows the effect of using NavigationStack with navigationTitle](https://raw.githubusercontent.com/Codecademy/docs/main/media/NavigationStack-with-navigationTitle.png)
+
+Another important modifier is `.navigationBarBackButtonHidden()`. This modifier removes the back button that appears when navigating to another view. This modifier is applied to the desired view:
+
+```Swift
+struct Doc: View {
+    let Names = ["Omar","Adam","Zack"]
+    var body: some View {
+        NavigationStack{
+            List(Names,id:\.self){ name in
+                NavigationLink {
+                    Text(name) // desired view destination
+                     .navigationBarBackButtonHidden()
+                } label: {
+                    Text(name)
+                }
+            }
+            .navigationTitle("Names")
+        }
+    }
+}
+```
+
+This code produces the following output: 
+
+![This gif shows the effect of using NavigationStack with navigationBarBackButtonHidden](https://raw.githubusercontent.com/Codecademy/docs/main/media/NavigationStack-with-navigationBarBackButtonHidden.gif)
```

Comment:
For all these gifs, can you crop out the safe areas?

Reply:
Nop I can't

Reply:
Do you need assistance with this?

Reply:
yes please I don’t know how to do that, because this gif is generated from the sim. I don’t have the capability to edit them. @yangc95